operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +470,283 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
                 .value(requestBaseURI + "/" + uriPath)
                 .build();
     }
+    
+    private void addIncludesSearchParameters(String compartmentMemberType, MultivaluedMap<String, String> searchParameters)
+    throws Exception {
+        // Add in Location, Medication, Organization, and Practitioner resources which are pointed to
+        // from search parameters only if the request does not have a _type parameter or it does have a 
+        // _type parameter that includes these
+
+        // Remove any _includes we added last time around
+        searchParameters.remove("_include");
+        
+        // Add in _includes for all search parameters that are Location, Medication, Organization, or Practitioner
+        if (compartmentMemberType.equals(AdverseEvent.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "location", Location.class.getSimpleName(), searchParameters);
```

Comment:
I think it'd be preferable to use ResourceType.Value.ADVERSE_EVENT instead of the getSimpleName.

Reply:
Done